azure-pipelines.yml

```diff
@@ -150,7 +150,7 @@ stages:
       - task: UseDotNet@2
         inputs:
           packageType: sdk
-          version: 5.0.100-preview.7.20366.6
+          version: 5.0.100-preview.8.20417.9
```

Comment:
Should `its\sources\Net5\global.json` be updated as well? There will be an impact on the GCP machines as well.

Reply:
> Should its\sources\Net5\global.json be updated as well?

It's interesting that it worked also with the old version since it was not installed.

> There will be an impact on the GCP machines as well.

Not sure what you mean. GCP machines are running a different build and they have different config. Sure, we should update also them but they are not related.

Reply:
Hmm, it shouldn't work. There must be some preview-aware logic in place.